src/seedu/addressbook/AddressBook.java

```diff
@@ -649,11 +649,21 @@ private static String getUserInput() {
         System.out.print(LINE_PREFIX + "Enter command: ");
         String inputLine = SCANNER.nextLine();
         // silently consume all blank and comment lines
-        while (inputLine.trim().isEmpty() || inputLine.trim().charAt(0) == INPUT_COMMENT_MARKER) {
+        while (isBlankLineOrComment(inputLine)) {
             inputLine = SCANNER.nextLine();
         }
         return inputLine;
     }
+    
+    /**
+     * Checks if input is a blank line or comment
```

Comment:
It is good that you have written the header comments for non-trivial methods. :)